### PR TITLE
Update Spring cloud api version

### PR DESCRIPTION
**Pull request by yuwzho**
**Task name**: Azure Spring Cloud

**Description**: Use stable API version and the preview API version is about to be retired.

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) N
**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Review by bowen5 — APPROVED** [review]

**Comment by yuwzho** [comment]
@chshrikh Can you help to take a look at this PR? This PR plans to update the API version to a stable one. The preview one is about to retired.

BTW, the [plugin's owner](https://github.com/microsoft/azure-pipelines-tasks/blob/182df2c072be73706d00241fcaa3c0d42c593c2c/.github/CODEOWNERS#L82) is no longer working on this.

**Comment by anatolybolshakov** [comment]
Tested changes with 'Set Production' action - looks fine for me, thanks! Changes looks good also.
@yuwzho could you please also bump task version to 2.198.0 - to reflect current sprint number?

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/AzureSpringCloudV0/task.loc.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
   "preview": true,
   "version": {
     "Major": 0,
-    "Minor": 185,
-    "Patch": 5
+    "Minor": 196,
```
Comment (EzzhevNikita) [inline Tasks/AzureSpringCloudV0/task.loc.json:21]: Please bump minor version up to 198

**Review by EzzhevNikita — APPROVED** [review]

File: Tasks/AzureSpringCloudV0/task.loc.json (hunk 1 above)
Comment (yuwzho) [reply Tasks/AzureSpringCloudV0/task.loc.json]: Updated

**Comment by anatolybolshakov** [comment]
Merging this one since we've checked task behavior and reviewed changes as well.

**Comment by anatolybolshakov** [comment]
Please note that there could be delays in roll out due to New Year deployment freeze.

**Comment by anatolybolshakov** [comment]
Hi @yuwzho PR changes has been already roll out, so it should be broadly available now.

**Comment by yuwzho** [comment]
> Hi @yuwzho PR changes has been already roll out, so it should be broadly available now.

Thank you! I can see that all the traffic is coming from the latest version

**Pull request merged**